extras/conf/gluster-exporter.toml.sample

```diff
@@ -8,6 +8,13 @@ metrics-path = "/metrics"
 log-dir = "/var/log/gluster-exporter"
 log-file = "exporter.log"
 log-level = "info"
+cache-ttl-in-sec = 30
+# by default all the functions are cached
+# functions which have to be disabled from caching should be added to 'cache-disabled-funcs'
+# supported functions are,
+# 'EnableVolumeProfiling', 'HealInfo', 'IsLeader', 'LocalPeerID', 'Peers',
+# 'Snapshots', 'VolumeBrickStatus', 'VolumeInfo', 'VolumeProfileInfo'
+cache-disabled-funcs = []
```

Comment:
VolumeProfileInfo is fine and its interval also for scraping might be increased. But EnableVolumeProfiling executes the command to enable only if its not already enabled so its fine. We need not enable caching for this as this function is not meant for scraping metrics data.

Reply:
I agree with @shtripat, we don't need caching for most of the functions except VolumeInfo, IsLeader. (LocalPeerID is also fine since it is just reading a local file on disk). VolumeProfileInfo is used by only one collector for which the interval is already made high.

Reply:
Acknowledged.